test/integration/rotation/04-create-workload-entry

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+log-debug "creating registration entry..."
+docker-compose exec spire-server \
+    /opt/spire/bin/spire-server entry create \
+    -parentID "spiffe://domain.test/spire/agent/x509pop/$(fingerprint conf/agent/agent.crt.pem)" \
+    -spiffeID "spiffe://domain.test/workload" \
+    -selector "unix:uid:0" \
+    -ttl 0
+
+# Wait up to 30 seconds for the agent to sync the entry
```

Comment:
I would add
PERCHECKSLEEP=1
here

or something to make this comment a bit more future proof

Reply:
👍